_data/projects/blockstreet.yml

```diff
@@ -0,0 +1,12 @@
+name: Blockstreet
+desc: Blockstreet aims to become the go-to source for the explanation of complex topics for users that are new to the blockchain ecosystem.
+site: https://blockstreet.io
+tags:
+- javascript
+- vuejs
+- nodejs
+- cryptocurrency
+- blockchain
+upforgrabs:
+  name: Status: Available
+  link: https://github.com/blockstreet/interface/issues
```

Comment:
This should now be `https://github.com/blockstreet/interface/labels/Status%3A%20Available`